pkg/config/config.go

```diff
@@ -37,6 +37,7 @@ func init() {
 	flag.StringVar(&Flags.ServiceAccountTokenPath, "service-account-token-path", "", "optionally override the default token path")
 	flag.StringVar(&Flags.MetricsAddr, "metrics-addr", "0.0.0.0:8081", "address to serve Prometheus metrics on")
 	flag.StringVar(&Flags.ProbeAddr, "probe-addr", "0.0.0.0:8080", "address to serve readiness/liveness probes on")
+	flag.StringVar(&Flags.OperatorNS, "operator-namespace", "kube-system", "namespace of the operator's k8s deployment")
```

Comment:
can we add a UT for this just to maintain coverage? also - if we maintain this, will the operator begin deploying to this ns in the overlay?

Reply:
re: UT - understandable makes sense

so part of rolling out this next release will entail feeding in the ccp ns as a command line argument?
I guess I'm just confused on what the intention is with this arg vs the NS arg

Reply:
NS arg is where things like external dns and nginx are deployed to. OperatorNS is where the operator is located. The two are completely different right now (app-routing-namespace vs kube-system). We need OperatorNS so we can set ownership references which are essential to garbage collection